components/Storyblok/partials/SbLink.tsx

```diff
@@ -1,22 +1,14 @@
 'use client';
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { config } from '@/utilities/config';
-import { cnb } from 'cnbuilder';
-import { useSearchParams } from 'next/navigation';
 import Link from 'next/link';
-import { SrOnlyText } from '@/components/Typography';
-import { type SbLinkType } from '../Storyblok.types';
+import { type SbLinkType } from '@/components/Storyblok/Storyblok.types';
 import { getMaskedAsset } from '@/utilities/getMaskedAsset';
+import useUTMs from '@/hooks/useUTMs';
 
-// TODO DS-1495: Let's see if we can clean this up a bit
 export type SbLinkProps = {
   link: SbLinkType;
-  classes?: string;
-  internalClasses?: string;
-  externalClasses?: string;
-  activeClass?: string;
-  assetClasses?: string;
-  hasExternalIcon?: boolean;
```

Comment:
Remove these specialized classes props from `SbLink`, renaming `classes` to `className` so it's consistent with our other components